engine/src/test/java/org/operaton/bpm/engine/impl/util/ImmutablePairTest.java

```diff
@@ -73,12 +74,9 @@ void shouldCompareWithRightSecond() {
   void shouldFailWithNonComparableTypes() {
     final ImmutablePair<Object, Object> pair1 = new ImmutablePair<>(new Object(), new Object());
     final ImmutablePair<Object, Object> pair2 = new ImmutablePair<>(new Object(), new Object());
-    try {
-      pair1.compareTo(pair2);
-      fail("Pairs should not be comparable");
-    } catch (IllegalArgumentException e) {
-      assertThat(e).hasMessageContaining("Please provide comparable elements");
-    }
+    assertThatThrownBy(() -> pair1.compareTo(pair2), "Pairs should not be comparable")
```

Comment:
Here is an example where the description could stay. Do you notice the difference?